app/src/main/res/values/styles.xml

```diff
@@ -129,4 +129,20 @@
         <item name="android:letterSpacing">0</item>
         <item name="android:textAllCaps">false</item>
     </style>
+
+    <style name="PoppinsRegular14" parent="@android:style/TextAppearance.Small">
```

Comment:
We shouldn't need those styles at all, I will ask our designer to use something we already have and if it will take too long to fix on her side we will need a task to fix this